benchmarks/c/miscellaneous/idd_dynamic.c

```diff
@@ -0,0 +1,43 @@
+#include <stdlib.h>
+#include <pthread.h>
+#include <assert.h>
+#include <stdatomic.h>
+
+// This test shows that data-dependencies have to be treated dynamically
+// Expected result: FAIL under ARM8 (possibly under other weak models as well)
+
+atomic_int x, y, z;
+
+void *thread_1(void *unused)
+{
+	int r = atomic_load_explicit(&x, memory_order_relaxed);
+	int s = atomic_load_explicit(&y, memory_order_relaxed);
+    int u = s;
```

Comment:
missing tab

Reply:
Had to enable whitespace rendering to see the difference locally.
Do we want to use actual tabs or regular spaces instead?
Any preferences about that? Or does it not matter as long as it is consistent in a single file?

Reply:
I don't have preferences about those, but we should be consistent.

That being said, the whole format of the project is a mess. If you check the diff of a PR (or even the content of a file from the web browser) the spacing format is all broken even if from the IDE looks nice (at least mine).

I will take a look if there is an easy way to automatically keep the format properly done without introducing much burden for development.